nitro-protocol/package.json

```diff
@@ -1,8 +1,11 @@
 {
-  "name": "contracts",
-  "version": "1.0.0",
+  "name": "@statechannels/nitro-protocol",
+  "version": "2.0.0-alpha.0",
   "description": "",
   "main": "index.js",
+  "files": [
+    "/contracts/**/*.sol"
+  ],
```

Comment:
We may wish to include some other stuff here, as well:
* typechain smart contract typings
* `solc` output (i.e. precompiled contracts ready for deployment)
* other typescript library code

We don't need to add those now, though.

Reply:
This will require deploying to NPM with a deployment script I think unless you want to add the build output to the repo

Reply:
Yes, that's right.